common/meta/raft-store/src/config.rs

```diff
@@ -98,6 +98,10 @@ pub struct RaftConfig {
 
     /// For test only: specifies the tree name prefix
     pub sled_tree_prefix: String,
+
+    ///  The node name. If the user specifies a name,
+    /// the user-supplied name is used, if not, the default name is used.
+    pub name: String,
```

Comment:
This should be the cluster name, not a name of a node in a cluster, IMHO

Reply:
Done.